packages/modeling/src/geometries/poly3/measureArea.js

```diff
@@ -14,19 +15,18 @@ const measureArea = (poly3) => {
   }
   const vertices = poly3.vertices
 
-  // calculate a real normal
-  const a = vertices[0]
```

Comment:
There was a reason for calculating the “real” normal, but I didn’t comment well.
Do all tests pass? Have you tried some exports to STL?

Reply:
All tests pass. Export works. In most cases, the vertex normal and the face normal will be the same (+/- epsilon). But I don't see how to fix the colinear case without something like this change.